pkg/system/phase1_verifying.go

```diff
@@ -275,25 +275,28 @@ func (r *Reconciler) checkBucketLoggingPVC() error {
 			"'Guaranteed' BucketLogging requires a Persistent Volume Claim (PVC) with ReadWriteMany (RWX) access mode. Please specify the 'BucketLoggingPVC' to ensure guaranteed logging")
 	}
 
-	// Check if pvc exists in the cluster
-	BucketLoggingPVC := &corev1.PersistentVolumeClaim{
-		TypeMeta: metav1.TypeMeta{Kind: "PersistenVolumeClaim"},
-		ObjectMeta: metav1.ObjectMeta{
-			Name:      *r.NooBaa.Spec.BucketLogging.BucketLoggingPVC,
-			Namespace: r.Request.Namespace,
-		},
-	}
-	if !util.KubeCheck(BucketLoggingPVC) {
-		return util.NewPersistentError("InvalidBucketLoggingConfiguration",
-			"The specified 'BucketLoggingPVC' was not found")
-	}
+	if r.NooBaa.Spec.BucketLogging.BucketLoggingPVC != nil {
+		// Check if pvc exists in the cluster
+		BucketLoggingPVC := &corev1.PersistentVolumeClaim{
+			TypeMeta: metav1.TypeMeta{Kind: "PersistenVolumeClaim"},
+			ObjectMeta: metav1.ObjectMeta{
+				Name:      *r.NooBaa.Spec.BucketLogging.BucketLoggingPVC,
+				Namespace: r.Request.Namespace,
+			},
+		}
+		if !util.KubeCheck(BucketLoggingPVC) {
+			return util.NewPersistentError("InvalidBucketLoggingConfiguration",
+				"The specified 'BucketLoggingPVC' was not found")
```

Comment:
As you are sending this patch, log name of the pvc also, *r.NooBaa.Spec.BucketLogging.BucketLoggingPVC.
On line 288 too.

Reply:
I agree let's add to the prints the name of the provided PVC, also in 298.

Reply:
Sure, make sense. Thanks